modules/efk-logging-kibana-interface-launch.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-kibana-interface.adoc
+
+[id="efk-logging-kibana-interface-launch-{context}"]
+= Launching the Kibana interface
+
+The Kibana interface is a browser-based console 
+to query, discover, and visualize your Elasticsearch data through histograms, line graphs, 
```

Comment:
to query, discover and visualize your logs through...  definitely dont want to promote it as a general data store